CPB_Ornament_Proximity/cpb_ornament_proximity.py

```diff
@@ -0,0 +1,101 @@
+"""
+Circuit Playground Bluefruit Ornament Proximity
+This demo uses advertising to set the color of scanning devices depending on the strongest broadcast
+signal received. Circuit Playgrounds can be switched between advertising and scanning using the
+slide switch. The buttons change the color when advertising.
+"""
+
+import time
+import board
+import digitalio
+
+import neopixel
+
+from adafruit_ble import BLERadio
+from adafruit_ble.advertising.adafruit import AdafruitColor
+
+# The color pickers will cycle through this list with buttons A and B.
+color_options = [0x110000,
+                 0x111100,
+                 0x001100,
+                 0x001111,
+                 0x000011,
+                 0x110011,
+                 0x111111,
+                 0x221111,
+                 0x112211,
+                 0x111122]
+
+ble = BLERadio()
+
+slide_switch = digitalio.DigitalInOut(board.SLIDE_SWITCH)
```

Comment:
please simplify by using the CircuitPlayground library? you can skip a bunch of this setup :)

Reply:
OK, I'll switch things over to CircuitPlayground library.

Reply:
@kattni recently added cpb support and supposedlyu should work just fine :) let us know if it does not

Reply:
Got loads of help from @kattni and now it works great using cpb!